components/runtime/wine/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2019 Evgeny Litvinenko
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		wine
+COMPONENT_VERSION=	4.4
+COMPONENT_REVISION=     1
```

Comment:
Initially, COMPONENT_REVISION should not be set.

Reply:
fixed